### DISCUSSION TITLE
CVE-2023-46809: Marvin attack on PKCS#1 padding

### DISCUSSION THREAD

**Discussion by Rubl80** [fuatakgun/eufy_security#1120] [General]
In the eufy-security-ws add-on log, I can see this error/security warning: CVE-2023-46809: Marvin attack on PKCS#1 padding
Wondering if anybody know if I should be worried or do anybody else experience the same behavior?

At the moment it does not affect the add-on and I have access to my eufy cameras in Home Assistant.
But earlier the connection to the devices was broken, but a restart of the add-on fixed that, which was why I noticed this warning in the first place.

- Logs from Eufy Security Add-on:
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
**SECURITY WARNING: Reverting CVE-2023-46809: Marvin attack on PKCS#1 padding**

**Comment by fuatakgun** [comment]
Will be resolved by bropat through add-on

**Reply by chrisgali1968** [reply]
This issue has been marked as closed in a corresponding entry (bropat)
